devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -21,16 +21,18 @@ metadata {
 		capability "Battery"
 		capability "Configuration"
 		capability "Refresh"
-		capability "Window Shade"
-		capability "Health Check"
+        capability "Health Check"
```

Comment:
nit: please use tabs for indentation in the whole file

Reply:
In the IDE, the indentation is correct, but when I commit it to Github, the indentation gets messed up. I've tried to fix this before, but it doesn't seem to work.

Reply:
Unfortunately it's tough to get the indentation right when using our IDE. I suggest copying the code into a different editor and fixing it up there.

Reply:
I have now fixed this.